### PR TITLE
Create reusable snackbar

### PR DESCRIPTION
**Pull request by Hristo-Boyadzhiev**
https://github.com/orgs/InterCarsBulgaria-TechLeadAcademy/projects/3/views/1?pane=issue&itemId=63164855

image

### CODE REVIEW

File: src/interfaces/snackbarContextValue.ts
Comment (vexleet) [inline src/interfaces/snackbarContextValue.ts:1]: such files should start with a capital letter.

File: src/hooks/useSnackbarContext.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+import { SnackbarContext } from '@/contexts/Snackbar'
+import { SnackbarContextValue } from '@/interfaces/snackbarContextValue'
+
+export const useSnackbarContext = (): SnackbarContextValue => {
```
Comment (vexleet) [inline src/hooks/useSnackbarContext.ts]: lets just write `useSnackbar` for this hook, same for the file name

Comment (Hristo-Boyadzhiev) [reply src/hooks/useSnackbarContext.ts]: Fixed

File: src/interfaces/snackbarContextValue.ts
Comment (Hristo-Boyadzhiev) [reply src/interfaces/snackbarContextValue.ts:1]: Fixed

**Review by vexleet — APPROVED** [review]

**Pull request merged**